modules/runner/src/test/ts/runner/HooksTest.ts

```diff
@@ -0,0 +1,66 @@
+import { HookType, Suite } from '@ephox/bedrock-common';
+import { assert } from 'chai';
+import { describe, it } from 'mocha';
```

Comment:
`mocha`?

Reply:
Mocha is what's used to test the bedrock logic, so this is expected.